pkg/rancher-desktop/config/settings.ts

```diff
@@ -308,7 +308,7 @@ export function runInDebugMode(debug: boolean): boolean {
   return debug || !!process.env.RD_DEBUG_ENABLED;
 }
 
-function fileExists(path: string) {
+export function fileExists(path: string) {
```

Comment:
It would be better to avoid calling this function because it's synchronous. See my comments where you call it.